02.calendar/cal.rb

```diff
@@ -0,0 +1,52 @@
+#!/Users/heureux/.rbenv/versions/2.6.5/bin/ruby
+
+require 'optparse'
+require 'date'
+
+days = ['日','月','火','水','木','金','土']
+
+opt = OptionParser.new
+opt.on('-y')
+opt.on('-m')
+opt.parse!(ARGV)
+
+if ARGV.empty?
+  targetYear = Date.today.year.to_i
+  targetMonth = Date.today.month.to_i
+else
+  arguments = ARGV
+  targetYear = arguments[0].to_i
+  targetMonth = arguments[1].to_i
+end
+
+lastDay = Date.new(targetYear, targetMonth, -1)
+lastDay = lastDay.day
+puts lastDay
+
+# カレンダータイトル
+puts "      #{targetYear}年#{targetMonth}月"
+
+# 曜日の表示
+days.each_with_index do |day, index|
+  if index == 6
+    puts "#{day} "
+  else
+    print "#{day} "
+  end
+end
```

Comment:
```suggestion
puts '日 月 火 水 木 金 土'
```

すいません、ちょっと伝わりにくかったかもしれませんが、前回の指摘事項は単純に↑のようにすればシンプルじゃない？という意図でした〜。

Reply:
@JunichiIto 確かにこの部分は変わることがないのでベタ書きにしました！